src/rgw/rgw_rados.h

```diff
@@ -1956,17 +1955,7 @@ class RGWRados
 
   RGWDataChangesLog *data_log;
 
-  virtual ~RGWRados() {
-    for (uint32_t i=0; i < num_rados_handles; i++) {
-      if (rados[i]) {
-        rados[i]->shutdown();
-        delete rados[i];
-      }
-    }
-    if (rados) {
-      delete[] rados;
-    }
-  }
+  virtual ~RGWRados() = default;
```

Comment:
We aren't calling `librados::Rados::shutdown()` anymore. Is that OK? Does `librados::Rados::~Rados()` do that for us?